tests/test_metadata_utils.py

```diff
@@ -1,22 +1,30 @@
 from unittest import TestCase
 from unittest.mock import Mock, MagicMock, patch
 
-from ebi_eva_common_pyutils.metadata_utils import resolve_variant_warehouse_db_name, get_taxonomy_code_from_taxonomy, \
-    get_assembly_code_from_assembly
+from ebi_eva_common_pyutils.metadata_utils import resolve_variant_warehouse_db_name, get_taxonomy_code_from_metadata, \
+    get_assembly_code_from_metadata, insert_new_assembly_and_taxonomy, build_taxonomy_code, \
+    ensure_taxonomy_is_in_evapro, insert_assembly_in_evapro, update_accessioning_status
 
 
 class TestMetadata(TestCase):
 
-    def test_get_taxonomy_code_from_taxonomy(self):
+    def test_build_taxonomy_code(self):
+        assert build_taxonomy_code('Homo sapiens') == 'hsapiens'
+        assert build_taxonomy_code('Triticum dicoccoides') == 'tdicoccoides'
+        assert build_taxonomy_code('Bison bonasus') == 'bbonasus'
+        # Not sure if we should be raising here
+        assert build_taxonomy_code('Homo') == 'h'
```

Comment:
Think it's probably ok for this one to be a "dumb" method and not raise anything...